app/Dashboard.html

```diff
@@ -53,9 +144,39 @@
                 "token":"${token}"
             }
           `
-        });
-        return false;
-      }
-    </script>
-  </body>
+    }).then(function (response) {
+        if (response.ok) {
+          toastr.success("Blog Post " + response.statusText);
+          form.reset();
+        } else {
+          toastr.warning("There was a problem submitting the blog post : " + response.statusText);
```

Comment:
We could probably change these to template strings - `\`There was a problem submitting the blog post :  ${response.statusText}\``

As far as JQuery, I _think_ Bootstrap requires JQuery, doesn't it? Not too much we can do to remove it as a dependency if that's the case